lib/Runtime/Base/ScriptContext.cpp

```diff
@@ -3998,7 +3998,7 @@ namespace Js
             OUTPUT_VERBOSE_TRACE(Js::DebuggerPhase, _u("DebugProfileProbeThunk: calling function: %s isWrapperRegistered=%d useDebugWrapper=%d\n"),
                 function->GetFunctionInfo()->HasBody() ? function->GetFunctionBody()->GetDisplayName() : _u("built-in/library"), AutoRegisterIgnoreExceptionWrapper::IsRegistered(scriptContext->GetThreadContext()), useDebugWrapper);
 
-            if (scriptContext->IsScriptContextInDebugMode())
+            if (scriptContext->IsScriptContextInDebugRecordingMode())
```

Comment:
>IsScriptContextInDebugRecordingMode [](start = 31, length = 35)

I think we don't need this method on ScriptContext. Just get the DebugContext and use the method on DebugContext as most of time we will be recording call and call StartRecordingCall